README.md

```diff
@@ -116,13 +117,27 @@ LAA_PORTAL_MOCK_SAML=false
 ```
 ** Note the above keys can be found in rattic
 
+### Authentication
+
 Mock Saml request on dev add the the following settings
 
 ```
 LAA_PORTAL_IDP_SSO_TARGET_URL=http://localhost:3002/saml/auth
 LAA_PORTAL_MOCK_SAML=true
 ```
 
+### Benefits checker
+
+To mock the benefits check in development and testing add the following environment
+variable:
+
+```
+USE_BC_DEV_MOCK=true
```

Comment:
I think the name `BC_USE_DEV_MOCK` would fit better with the other benefit checker settings : 
```
BC_LSC_SERVICE_NAME
BC_CLIENT_ORG_ID
BC_CLIENT_USER_ID
BC_WSDL_URL
```